lib/tasks/publishing_api.rake

```diff
@@ -0,0 +1,50 @@
+namespace :publishing_api do
+  desc "Publish special routes such as the homepage"
+  task :publish_special_routes => :environment do
+    require 'gds_api/publishing_api/special_route_publisher'
+
+    publisher = GdsApi::PublishingApi::SpecialRoutePublisher.new
+
+    routes = [
+      {
+        content_id: "f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a",
+        base_path: "/",
+        title: "GOV.UK homepage",
+      },
+      {
+        content_id: "ffcd9054-ee77-4539-978d-171a60eb4b2a",
```

Comment:
Yeah no magic, just a copy pasta mistake.

I'll add the redirect idea to the list of things to review when all these PRs are ready.  We've another 7 to do.